src/kibana/components/vislib/styles/_tilemap.less

```diff
@@ -79,16 +79,52 @@
 }
 
 .leaflet-popup-content-wrapper {
-  background: rgba(70, 82, 93, 0.95) !important;
-  color: @gray-lighter !important;
+  background: @tooltip-bg !important;
+  color: @tooltip-color !important;
   border-radius: 4px !important;
   padding: 0 !important;
 }
 
 .leaflet-popup-content {
-  padding: 8px !important;
   margin: 0 !important;
-  line-height: 14px !important;
+  line-height: 1.1 !important;
+  font-size: 12px;
```

Comment:
Tooltip font styling should be abstracted into variables that are used in both this tooltip and the normal ones.